### DISCUSSION TITLE
Bring back key presses as substitute for mouse clicks in menus

### DISCUSSION THREAD

**Discussion by OhCanada1867** [ppy/osu#28345] [Ideas] [answered]
In the Stable client during gameplay, you can use the keys (z/x or whatever you have set) instead of mouse clicks to interact with the pause menu to continue, retry, etc and it has become a staple of how I navigate the game. In Lazer however, this is not implemented and I feel that adding this would drastically improve the flow of navigation. Additionally, I believe the key presses should be able to be used in the main menu to select single player, multiplayer, etc, a feature not present in stable as other shortcuts that could not be changed were used instead.

Thank you for your consideration and keep on doing what you're doing dev team <3

**Comment by Stoppedpuma** [comment, marked as answer]
Duplicate of #3551
